specs/latest/2.0/index.html

```diff
@@ -2809,6 +2809,20 @@ <h2 class="no-toc">Table of contents</h2>
         There is no use case for these parameters being different.
     </div>
 
+    <p>
+        If a fragment shader writes to none of <code>gl_FragColor</code>, <code>gl_FragData</code>, nor any
+        user-defined output variables, the values of the fragment colors following shader execution are untouched.
+        If some, but not all user-defined output variables are written, the values of fragment colors corresponding
+        to unwritten variables are similarly untouched.
+    </p>
+
+    <p>
+        If the values written by the fragment shader do not match the format(s) of the corresponding
+        color buffer(s), and the color buffer states are not set to <code>NONE</code> by <code>DrawBuffers</code>,
```

Comment:
Is it worth adding some non-normative text with an example of this mismatch? For example, clearly stating that the shader attempts to write an integer value to a floating-point color buffer, or vice versa?